### DISCUSSION TITLE
Generate fixed documentation for production

### DISCUSSION THREAD

**Discussion by LvckyAPI** [dedoc/scramble#625] [Ideas]
The docs are currently regenerated each time they are called up.

If you write a public API, this may be unfavorable.
If you now make the API docs public and hundreds of users access them, the server will be DDoSed in no time at all.

It would therefore be practical if you could somehow store a ready-generated version for production to save resources.

These new generations can then be renewed via CI/CD pipeline during deployment.

In local use, however, constant re-generation is not a problem.

**Comment by romalytvynenko** [comment]
@LvckyAPI there is a command, `php artisan scramble:export` that will generate `api.json` file with the spec. The file itself then can be included in some UI lib for rendering it
